opentelemetry-example-app/src/opentelemetry_example_app/metrics_example.py

```diff
@@ -11,26 +11,32 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
+"""
+This module serves as an example for a simple application using metrics
+"""
 
-# pylint: skip-file
 from opentelemetry import metrics
+from opentelemetry.sdk.metrics import Meter
 
-METER = metrics.Meter()
-MEASURE = metrics.create_measure(
-    "idle_cpu_percentage",
-    "cpu idle over time",
-    "percentage",
-    metrics.ValueType.FLOAT,
-    ["environment"],
+metrics.set_preferred_meter_implementation(lambda _: Meter())
```

Comment:
Any reason the preferred implementation shouldn't be the default SDK if the user doesn't specify otherwise? For both `Meter` and `Tracer`. I remember some discussion around the time of #29, but I don't see a good argument there against this.

My assumption is that most users will use the default implementation, so we should let the minority opt out by loading a different one instead of forcing the majority to opt-in. I can imagine this bit of clunky boilerplate turning people off of the library if this is the first example they're exposed to.

Another problem with the current approach is that this loads the _API_ `Meter` class by default, which is effectively an interface. Since that `Meter` returns null for every method, anyone unlucky enough to load it accidently is going to run into an NPE pretty quickly. A true no-op implementation should return dummy objects of the expected type instead.

Reply:
--> #165,  open-telemetry/opentelemetry-specification#171

Reply:
I believe the loading the default SDK is something we decided against in previous discussions. Will bring it up in the SIG. As for a no-op implementation, I will add this.